drivers/uktty/pl011/Config.uk

```diff
@@ -5,13 +5,8 @@ menuconfig LIBUKTTY_PL011
 	select LIBUKOFW
 
 if LIBUKTTY_PL011
+
 menuconfig LIBUKTTY_PL011_EARLY_CONSOLE
 	bool "Early console"
 
-if LIBUKTTY_PL011_EARLY_CONSOLE
-config LIBUKTTY_PL011_EARLY_CONSOLE_BASE
```

Comment:
Similar to my comment to ns16550: Do we really want to remove option to compile in a default for ultra minimal builds without fdt and without kernel command line parameters?

Reply:
Answered in https://github.com/unikraft/unikraft/pull/1473#discussion_r1726144448